LiskHQ/lisk-explorer
# Redis cache error - Closes #598

### What was the problem?
Redis warning message was showing when trying to get an invalid api call

### How did I fix it?
Added a check before saving the response

### How to test it?
Access `/api/invalid/url`
You should see `Cannot GET /api/invalid/url` on the browser
You should NOT see redis warning on server console

### Review checklist
- The PR solves #598 
- All new code is covered with unit tests
- All new features are covered with e2e tests
- All new code follows best practices
